wit/deps/wasm-rpc/wasm-rpc.wit

```diff
@@ -97,13 +97,19 @@ interface types {
 
     /// Schedule invocation for later
     schedule-invocation: func(scheduled-time: datetime, function-name: string, function-params: list<wit-value>) -> ();
```

Comment:
Left the old one for backwards compat as we already cut a release.
In principle we can have a more effecient implementation for the non cancellation version as it's idempotent.

(This version would be like the old code, currently I implemented it in terms of the new function to reduce code size)